src/bicep/examples/deploymentVariables.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
Please remove this file because it's now excluded in the `.gitignore`.

Reply:
Done.